edx_arch_experiments/kafka_consumer/docs/decisions/0001_kafka_based_event_bus.rst

```diff
@@ -0,0 +1,154 @@
+=====================
+Kafka-Based Event Bus
+=====================
+
+Status
+------
+
+Provisional
+
+Context
+-------
+
+The draft `OEP-52: Event Bus Architecture`_ explains how the Open edX platform would benefit from an event bus, as well as providing some additional decisions around the event bus. One decision is to enable the event bus technology to be pluggable through some abstraction layer.
+
+However, an initial technology, even if multiple could eventually be selected, is still required.
```

Comment:
The construction of this sentence is a little hard to follow

```suggestion
This still requires selecting specific technology for the first implementation.
```

Reply:
Thanks. I'll use this, but add `a` before `specific technology`.